abfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    settings.solvent_lambda_settings.lambda_elec # List of floats of lambda values for the electrostatics for the solvent leg
</pre><p>add where 0.0 is fully interacting and 1.0 is off</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>